src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_BinaryOperator.cs

```diff
@@ -1859,6 +1860,25 @@ private bool TryGetNullableMethod(SyntaxNode syntax, TypeSymbol nullableType, Sp
             return false;
         }
 
+        /// <summary>
+        /// Tries to get member of a nullable value type in a soft manner (without reporting an error if member is not found).
+        /// Prefer this when doing optimizations, so if a special member is present, compiler generates more optimal code,
+        /// but if not, just gracefully fallbacks to less optimal version without reporting unnecessary errors
+        /// </summary>
+        private bool TryGetNullableMethodSoft(TypeSymbol nullableType, SpecialMember specialMember, [NotNullWhen(true)] out MethodSymbol? result)
```

Comment:
>TryGetNullableMethodSoft [](http://example.com/codeflow?start=21&length=24)

I recommend to not "reinvent a wheel" and use one of the existing helpers. The Binder probably already has a helper with desired behavior